pxr/base/tf/unicodeUtils.h

```diff
@@ -94,6 +170,18 @@ class TfUtf8CodePointView final {
         return _view.empty();
     }
 
+    /// Returns an iterator of the same type as begin that identifies the end
```

Comment:
Optional, but it might help readability to put "begin" in code-font or italics. Maybe also applies to "PastTheEndSentinel" below.